CS342MemberViews/Sources/AndyWang/HobbiesView.swift

```diff
@@ -0,0 +1,72 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+import SwiftUI
+
+
+struct HobbiesView: View {
+    var body: some View {
+        ScrollView(.vertical) {
+            VStack(spacing: 30) {
+                Text("- I am the alto saxophonist for the Stanford Saxophone Quartet, and we host quarterly recitals and performances on campus.")
+                saxophoneImage
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(maxWidth: 300)
+                Text("- I also enjoy playing volleyball! Currently I play on the Stanford Club Men's Volleyball Team as an outside hitter.")
+                volleyballImage
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(maxWidth: 300)
+                Text("- A more minor hobby of mine is video editing! I use Adobe Premiere for small personal projects here and there.")
+                premiereImage
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(maxWidth: 150)
+            }
+            .padding(20)
+        }
+            .navigationTitle("My Hobbies!")
+    }
+    
+    
+    private var saxophoneImage: Image {
+        guard let imagePath = Bundle.module.path(forResource: "Saxophone", ofType: "jpg"),
```

Comment:
There is actually an extension on `Bundle` we defined for you in the `Bundle+Image.swift` file that you can use instead of rewriting this code for each image. 

You can use it to reference this image as follows:

```swift
Bundle.module.image(fromFileNamed: "Saxophone", type: "jpg")
```

Which is much simpler!

Reply:
Thank you for the feedback! Is it ok if I leave the code as-is for this assignment? Or should I modify my code for a regrade as Paul mentioned in lecture?